modules/api/src/main/java/org/eclipse/xpanse/api/config/ServiceTemplateEntityConverter.java

```diff
@@ -37,10 +37,11 @@ public static ServiceTemplateDetailVo convertToServiceTemplateDetailVo(
             serviceTemplateDetailVo.setServiceTemplateId(serviceTemplateEntity.getId());
             serviceTemplateDetailVo.setIcon(serviceTemplateEntity.getOcl().getIcon());
             serviceTemplateDetailVo.setDescription(serviceTemplateEntity.getOcl().getDescription());
-            if (StringUtils.isNotEmpty(serviceTemplateEntity.getNamespace())) {
-                serviceTemplateDetailVo.setNamespace(serviceTemplateEntity.getNamespace());
+            if (StringUtils.isNotEmpty(serviceTemplateEntity.getServiceVendor())) {
+                serviceTemplateDetailVo.setServiceVendor(serviceTemplateEntity.getServiceVendor());
             } else {
```

Comment:
The if... else... code is unnecessary, the value already has been set by called BeanUtils.copyProperties(serviceTemplateEntity, serviceTemplateDetailVo) .